assets/sass/modules/_enroll.scss

```diff
@@ -350,6 +343,10 @@ $enroll-content-spacing: 15px;
     position: relative;
   }
 
+  .manual-setup {
+    font-size: 13px;
```

Comment:
why do we need to change the font size? Is the default one not right? A form title, subtitle and content already have default values, so if we can stick to those I think it is better in the long term

Reply:
By default anchor has 14px. In this case UX wants it to be 13px.